types/npm-registry-fetch/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for npm-registry-fetch 4.0
+// Type definitions for npm-registry-fetch 8.0
```

Comment:
Will this break anyone still using older versions? When upgrading the major version, please consider the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#if-a-library-is-updated-to-a-new-major-version-with-breaking-changes-how-should-i-update-its-type-declaration-package

Reply:
It looks like this would break the types for pacote v9. I have another change to update the types for pacote that would fix that, but I found that both npm-registry-fetch and pacote have tags for maintaining these old versions, so I'll update this to match.